libsolidity/formal/ModelCheckerSettings.h

```diff
@@ -75,6 +99,7 @@ struct ModelCheckerTargets
 
 struct ModelCheckerSettings
 {
+	ModelCheckerContracts contracts = ModelCheckerContracts::All();
 	ModelCheckerEngine engine = ModelCheckerEngine::None();
 	ModelCheckerTargets targets = ModelCheckerTargets::All();
```

Comment:
Nit: I think we should rename both `All()` to `Default()` or something like that.